### PR TITLE
create static-pages

### PR DESCRIPTION
**Pull request by PhamVanTung**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: ` full_title yield(:title)`

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ''
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
+  end
```
Comment (nguyenduyta) [inline app/helpers/application_helper.rb]: page_title.empty? ? "#{base_title}" : "#{page_title}  |  #{base_title}"

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
```
Comment (nguyenduyta) [inline app/views/layouts/_footer.html.erb]: indent

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "TEST APP", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",    root_path %></li>
+        <li><%= link_to "Help",    help_path %></li>
```
Comment (nguyenduyta) [inline app/views/layouts/_header.html.erb]: indent

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,14 +1,22 @@
...
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <%= stylesheet_link_tag 'application', media: 'all',
+                                           'data-turbolinks-track' => true %>
```
Comment (nguyenduyta) [inline app/views/layouts/application.html.erb]: indent
exchange: data-turbolinks-track: true

**Pull request merged**